src/stylesheets/layouts/_header.scss

```diff
@@ -2,44 +2,50 @@
   background-color: $color-mint;
   display: flex;
   justify-content: center;
+  position: relative;
 
   &__wrapper {
-    margin: 50px;
-    // position: relative;
+    max-width: 50rem;
+    margin: 5rem 6rem;
   }
-  // &__primary {
-  //   display: flex;
-  //   justify-content: center;
-  //
-  //   &--top {
-  //     display: block;
-  //     margin-left: auto;
-  //     margin-right: auto;
-  //     text-align: center;
-  //   }
-  //
-  //   &--btm {
-  //     display: block;
-  //     width: 500px;
-  //   }
-  // }
-  &__text-box {
+
+  &__primary {
+    width: 100%;
+    display: block;
+    position: relative;
+
+    &--top {
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+      text-align: center;
```

Comment:
you shouldn't need both the auto margins and the `text-align: center`. One or the other should do the job. I'd keep the `text-align: center` since this would need a width or max-width for the auto margins to do anything.